src/renderer.ts

```diff
@@ -363,18 +398,36 @@ const render = (format: Format, imageMappings: ImageMappings) =>
 
             return h('div', props);
         }
-    }
+
+        case ElementKind.ContentAtom:
+            if (element.atom.atomType === AtomType.INTERACTIVE) {
```

Comment:
Does this only apply to atoms that appear on full-page interactives? Or is it any interactive atom, including ones that appear in standard articles?

Reply:
This will appear on other article types too I think. 

Due to the API we'll need to handle the other types differently, and will have to make some new components:

`const html = element.atom.data.recipe?.html`

Reply:
Perhaps we'll come up with a new data type for atoms we support:

```
const enum AtomKind {
    Interactive,
    Recipe,
}

interface Atom {
    kind: AtomKind;
    js: string;
    css: string;
    html: string;
}
```